recipe/build.sh

```diff
@@ -1,11 +1,19 @@
 #!/bin/bash
+# Get an updated config.sub and config.guess
+cp $BUILD_PREFIX/share/gnuconfig/config.sub .
+cp $BUILD_PREFIX/share/gnuconfig/config.guess .
 
 if [ `uname` == Linux ]; then
     # Enable -z def linked flag
     # linking will fail when undefined symbols are present
     export CFLAGS="${CFLAGS} -Wl,-z,defs"
+elif [ `uname` == osx ]; then
+    export CFLAGS="${CFLAGS} --std=c90,-Wno-implicit-function-declaration,-z"
+    export L="/usr/local/lib"
```

Comment:
can we try removing these 3 lines?

Reply:
Yeah we can try removing those! Thank you for taking a look at this! I'll have computer access to remove those soon